platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -72,6 +72,9 @@ RUN pip install crontab
 RUN pip install xmltodict==0.12.0
 RUN pip install jsondiff==1.2.0
 RUN pip install ijson==2.6.1
+RUN pip3 install xmltodict==0.12.0
+RUN pip3 install jsondiff==1.2.0
+RUN pip3 install ijson==2.6.1
```

Comment:
Can you add these dependencies to the `install_requires=` section of the setup.py file so that they get installed implicitly instead? We should then also no longer need to explicitly install them in the slave containers, either.

Reply:
@jleveque : I was doing this change, but how to handle build or test requirements. I notice that since ijson is not part of slave docker, it fails while build time tests.  i added build and test requirements but does not pass.

```
OK
running bdist_wheel
running build
running build_py
Traceback (most recent call last):
  File "./tests/yang_model_tests/yangModelTesting.py", line 7, in <module>
    import ijson
ImportError: No module named ijson
YANG Tests failed
```

Reply:
Is it only needed by the tests? If so, adding the line to `tests_require=` should be sufficient for setuptools to install it before running tests.

Reply:
I added tests_require etc, but I still see build failure without package being installed in slave docker. I raised the commit let's see if it passes on Jenkins. Let me know from commit review if I am missing anything.

Reply:
Done now,